shaders/src/post_process_old.fs

```diff
@@ -4,11 +4,11 @@ LAYOUT_LOCATION(0) out vec4 fragColor;
 
 #if POST_PROCESS_TONE_MAPPING
 vec3 resolveFragment(const ivec2 uv) {
-    return texelFetch(postProcess_colorBuffer, uv, 0).rgb;
+    return vec3(0.0);
```

Comment:
This whole file will be removed in a future PR, these changes just gets it to build.